client/api/two/webhooks.py

```diff
@@ -0,0 +1,48 @@
+from client.resource import Resource
+
+
+class Webhooks(Resource):
+
+    def all(self, page=None, limit=20):
+        params = {
+            'page': page,
+            'limit': limit
+        }
+        return self.request_get('webhooks', params)
+
+    def create(self, event, target, conditions, enabled=None):
+        data = {
+            'event': event,
+            'target': target,
+            'conditions': conditions
+        }
+        if enabled is not None:
+            data.update({
+                'enabled': enabled
+            })
+        return self.request_post('webhooks', data)
+
+    def get(self, webhook_id):
+        return self.request_get('webhooks/{}'.format(webhook_id))
+
+    def update(
```

Comment:
can you change the args and kwargs so they are not all in a new line, you can do
```
def update(
    self, webhook_id, event=None, target=None, conditions=None,
    enabled=None
):
````

or [check style guide](https://www.python.org/dev/peps/pep-0008/#indentation) in case it's too long and linter whines about it.

Reply:
Sure, will do.